### DISCUSSION TITLE
Is it possible / safe to cancel a running backup?

### DISCUSSION THREAD

**Discussion by msullivan-lc** [nextcloud/all-in-one#5287] [Questions] [answered]
Wondering if it is possible and/or safe to cancel a running backup started from the AIO interface?

**Comment by msullivan-lc** [comment]
Follow-up question. I had a backup fail due to not enough space on the backup drive. When the backup failed, about 18G of space was freed up on the drive that the backup was being made to, is there an easy way to remove or purge the rest of the failed backup contents. Prior to the failed backup, the drive had about 380G free.

**Comment by szaimen** [comment, marked as answer]
> Wondering if it is possible and/or safe to cancel a running backup started from the AIO interface?

It depends. Why do you want to do this?

> Follow-up question. I had a backup fail due to not enough space on the backup drive. When the backup failed, about 18G of space was freed up on the drive that the backup was being made to, is there an easy way to remove or purge the rest of the failed backup contents. Prior to the failed backup, the drive had about 380G free.

Usually if you do not stop the container manually, it should clean up failed backups automatically.

**Reply by msullivan-lc** [reply]
> It depends. Why do you want to do this?

I accidentally started the backup without realizing it would be too large for my available disk space.

> Usually if you do not stop the container manually, it should clean up failed backups automatically.

So in this case, I didn't stop the container manually, it did throw messages in the log about failing due to no drive space available. As noted, it only cleaned up about 18GB. I'm considering moving the entire borg folder to a disk with more space then making a symlink to the new location from the old location since there isn't any to change the backup path.

Another question: is there a way to reset the backup system entirely outside of resetting the entire installation? A way to delete all backups and start over would be great!

**Reply by szaimen** [reply]
You can simply create the target folder via CLI

**Reply by msullivan-lc** [reply]
After installing the borgbackup utility on the box, I was able to get some info on the repo. It was only listing the successfully completed backups in the list command, but filtering for --last 1 showed the failed backup. 

I ran borg delete "repo" --last 1 then borg compact "repo" and that cleaned up the files for the failed backup.

For now, my symlink also seems to be working.

Thanks!
